converters/google/convert.go

```diff
@@ -173,33 +173,43 @@ type Converter struct {
 func (c *Converter) AddResourceChanges(changes []*tfjson.ResourceChange) error {
 	var createOrUpdateOrNoops []*tfjson.ResourceChange
 	for _, rc := range changes {
-		// skip unknown resources
+		// Silently skip non-google resources
+		if !strings.HasPrefix(rc.Type, "google_") {
+			continue
+		}
+
+		// Warn about google-beta resources
+		if rc.ProviderName == "registry.terraform.io/hashicorp/google-beta" {
+			c.errorLogger.Debug(fmt.Sprintf("%s: resource uses the google-beta provider and may not be convertable", rc.Address))
```

Comment:
Nit: Not sure if there is precedence with "convertable", but I believe it technically should be "convertible"

Reply:
nope, no precedent. just misspelled it. fixed in 414602ab